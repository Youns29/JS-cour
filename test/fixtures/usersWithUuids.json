[
  {
    "slug": "bbdf7566-234b-4b51-a50d-12e5d16258ea",
    "name": "Leffler",
    "firstName": "Herman",
    "note": 9
  },
  {
    "slug": "d53ce32f-ca6a-475c-9bb2-0736e16b558c",
    "name": "Altenwerth",
    "firstName": "Toni",
    "note": 13
  },
  {
    "slug": "d53ce32f-ca6a-475c-9bb2-0736e16b558c",
    "name": "Altenwerth",
    "firstName": "Toni",
    "note": 19
  },
  {
    "slug": "7e5cc890-679f-4510-9dd8-d823f05c2dc3",
    "name": "Ernser",
    "firstName": "Julio",
    "note": 10
  },
  {
    "slug": "05f98060-71c8-4bc8-9941-d9e1ff22a709",
    "name": "Ledner",
    "firstName": "Leroy",
    "note": 19
  },
  {
    "slug": "7e5cc890-679f-4510-9dd8-d823f05c2dc3",
    "name": "Ernser",
    "firstName": "Julio",
    "note": 3
  },
  {
    "slug": "382f4d43-8e53-47d4-acac-71bafbfe2a2c",
    "name": "Willms",
    "firstName": "Carolyn",
    "note": 3
  },
  {
    "slug": "d53ce32f-ca6a-475c-9bb2-0736e16b558c",
    "name": "Altenwerth",
    "firstName": "Toni",
    "note": 10
  },
  {
    "slug": "f5d26aed-0f48-43d8-89f3-c9f4dcfb7b9d",
    "name": "Schamberger",
    "firstName": "Clifton",
    "note": 1
  },
  {
    "slug": "6138bc99-6599-42c1-a916-fb61d0672284",
    "name": "Lehner",
    "firstName": "Alonzo",
    "note": 11
  },
  {
    "slug": "6138bc99-6599-42c1-a916-fb61d0672284",
    "name": "Lehner",
    "firstName": "Alonzo",
    "note": 6
  },
  {
    "slug": "d53ce32f-ca6a-475c-9bb2-0736e16b558c",
    "name": "Altenwerth",
    "firstName": "Toni",
    "note": 16
  },
  {
    "slug": "c3454c44-036b-4b40-ae0d-476dd396304d",
    "name": "Goldner",
    "firstName": "Hope",
    "note": 12
  },
  {
    "slug": "05f98060-71c8-4bc8-9941-d9e1ff22a709",
    "name": "Ledner",
    "firstName": "Leroy",
    "note": 4
  },
  {
    "slug": "f5d26aed-0f48-43d8-89f3-c9f4dcfb7b9d",
    "name": "Schamberger",
    "firstName": "Clifton",
    "note": 15
  },
  {
    "slug": "05f98060-71c8-4bc8-9941-d9e1ff22a709",
    "name": "Ledner",
    "firstName": "Leroy",
    "note": 12
  },
  {
    "slug": "6138bc99-6599-42c1-a916-fb61d0672284",
    "name": "Lehner",
    "firstName": "Alonzo",
    "note": 20
  },
  {
    "slug": "f5d26aed-0f48-43d8-89f3-c9f4dcfb7b9d",
    "name": "Schamberger",
    "firstName": "Clifton",
    "note": 19
  },
  {
    "slug": "c3454c44-036b-4b40-ae0d-476dd396304d",
    "name": "Goldner",
    "firstName": "Hope",
    "note": 4
  },
  {
    "slug": "f5d26aed-0f48-43d8-89f3-c9f4dcfb7b9d",
    "name": "Schamberger",
    "firstName": "Clifton",
    "note": 16
  },
  {
    "slug": "7e5cc890-679f-4510-9dd8-d823f05c2dc3",
    "name": "Ernser",
    "firstName": "Julio",
    "note": 1
  },
  {
    "slug": "05f98060-71c8-4bc8-9941-d9e1ff22a709",
    "name": "Ledner",
    "firstName": "Leroy",
    "note": 13
  },
  {
    "slug": "382f4d43-8e53-47d4-acac-71bafbfe2a2c",
    "name": "Willms",
    "firstName": "Carolyn",
    "note": 17
  },
  {
    "slug": "6138bc99-6599-42c1-a916-fb61d0672284",
    "name": "Lehner",
    "firstName": "Alonzo",
    "note": 10
  },
  {
    "slug": "f5d26aed-0f48-43d8-89f3-c9f4dcfb7b9d",
    "name": "Schamberger",
    "firstName": "Clifton",
    "note": 20
  },
  {
    "slug": "382f4d43-8e53-47d4-acac-71bafbfe2a2c",
    "name": "Willms",
    "firstName": "Carolyn",
    "note": 0
  },
  {
    "slug": "4e74a047-70b5-4dde-8caa-d41c62f64e83",
    "name": "Keeling",
    "firstName": "Patty",
    "note": 4
  },
  {
    "slug": "c3454c44-036b-4b40-ae0d-476dd396304d",
    "name": "Goldner",
    "firstName": "Hope",
    "note": 16
  },
  {
    "slug": "382f4d43-8e53-47d4-acac-71bafbfe2a2c",
    "name": "Willms",
    "firstName": "Carolyn",
    "note": 17
  },
  {
    "slug": "382f4d43-8e53-47d4-acac-71bafbfe2a2c",
    "name": "Willms",
    "firstName": "Carolyn",
    "note": 12
  },
  {
    "slug": "f5d26aed-0f48-43d8-89f3-c9f4dcfb7b9d",
    "name": "Schamberger",
    "firstName": "Clifton",
    "note": 6
  },
  {
    "slug": "6138bc99-6599-42c1-a916-fb61d0672284",
    "name": "Lehner",
    "firstName": "Alonzo",
    "note": 7
  },
  {
    "slug": "4febcf39-0304-4102-8cdf-dce4412f2da9",
    "name": "Bode",
    "firstName": "Audrey",
    "note": 20
  },
  {
    "slug": "7e5cc890-679f-4510-9dd8-d823f05c2dc3",
    "name": "Ernser",
    "firstName": "Julio",
    "note": 2
  },
  {
    "slug": "6138bc99-6599-42c1-a916-fb61d0672284",
    "name": "Lehner",
    "firstName": "Alonzo",
    "note": 13
  },
  {
    "slug": "f5d26aed-0f48-43d8-89f3-c9f4dcfb7b9d",
    "name": "Schamberger",
    "firstName": "Clifton",
    "note": 14
  },
  {
    "slug": "c3454c44-036b-4b40-ae0d-476dd396304d",
    "name": "Goldner",
    "firstName": "Hope",
    "note": 13
  },
  {
    "slug": "bbdf7566-234b-4b51-a50d-12e5d16258ea",
    "name": "Leffler",
    "firstName": "Herman",
    "note": 19
  },
  {
    "slug": "7e5cc890-679f-4510-9dd8-d823f05c2dc3",
    "name": "Ernser",
    "firstName": "Julio",
    "note": 18
  },
  {
    "slug": "7e5cc890-679f-4510-9dd8-d823f05c2dc3",
    "name": "Ernser",
    "firstName": "Julio",
    "note": 13
  },
  {
    "slug": "05f98060-71c8-4bc8-9941-d9e1ff22a709",
    "name": "Ledner",
    "firstName": "Leroy",
    "note": 15
  },
  {
    "slug": "4e74a047-70b5-4dde-8caa-d41c62f64e83",
    "name": "Keeling",
    "firstName": "Patty",
    "note": 5
  },
  {
    "slug": "d53ce32f-ca6a-475c-9bb2-0736e16b558c",
    "name": "Altenwerth",
    "firstName": "Toni",
    "note": 8
  },
  {
    "slug": "7e5cc890-679f-4510-9dd8-d823f05c2dc3",
    "name": "Ernser",
    "firstName": "Julio",
    "note": 18
  },
  {
    "slug": "7e5cc890-679f-4510-9dd8-d823f05c2dc3",
    "name": "Ernser",
    "firstName": "Julio",
    "note": 19
  },
  {
    "slug": "6138bc99-6599-42c1-a916-fb61d0672284",
    "name": "Lehner",
    "firstName": "Alonzo",
    "note": 13
  },
  {
    "slug": "c3454c44-036b-4b40-ae0d-476dd396304d",
    "name": "Goldner",
    "firstName": "Hope",
    "note": 8
  },
  {
    "slug": "05f98060-71c8-4bc8-9941-d9e1ff22a709",
    "name": "Ledner",
    "firstName": "Leroy",
    "note": 5
  },
  {
    "slug": "7e5cc890-679f-4510-9dd8-d823f05c2dc3",
    "name": "Ernser",
    "firstName": "Julio",
    "note": 18
  },
  {
    "slug": "bbdf7566-234b-4b51-a50d-12e5d16258ea",
    "name": "Leffler",
    "firstName": "Herman",
    "note": 9
  },
  {
    "slug": "bbdf7566-234b-4b51-a50d-12e5d16258ea",
    "name": "Leffler",
    "firstName": "Herman",
    "note": 12
  },
  {
    "slug": "4febcf39-0304-4102-8cdf-dce4412f2da9",
    "name": "Bode",
    "firstName": "Audrey",
    "note": 3
  },
  {
    "slug": "4e74a047-70b5-4dde-8caa-d41c62f64e83",
    "name": "Keeling",
    "firstName": "Patty",
    "note": 15
  },
  {
    "slug": "382f4d43-8e53-47d4-acac-71bafbfe2a2c",
    "name": "Willms",
    "firstName": "Carolyn",
    "note": 18
  },
  {
    "slug": "d53ce32f-ca6a-475c-9bb2-0736e16b558c",
    "name": "Altenwerth",
    "firstName": "Toni",
    "note": 14
  },
  {
    "slug": "c3454c44-036b-4b40-ae0d-476dd396304d",
    "name": "Goldner",
    "firstName": "Hope",
    "note": 20
  },
  {
    "slug": "d53ce32f-ca6a-475c-9bb2-0736e16b558c",
    "name": "Altenwerth",
    "firstName": "Toni",
    "note": 19
  },
  {
    "slug": "4febcf39-0304-4102-8cdf-dce4412f2da9",
    "name": "Bode",
    "firstName": "Audrey",
    "note": 7
  },
  {
    "slug": "f5d26aed-0f48-43d8-89f3-c9f4dcfb7b9d",
    "name": "Schamberger",
    "firstName": "Clifton",
    "note": 10
  },
  {
    "slug": "05f98060-71c8-4bc8-9941-d9e1ff22a709",
    "name": "Ledner",
    "firstName": "Leroy",
    "note": 3
  },
  {
    "slug": "4e74a047-70b5-4dde-8caa-d41c62f64e83",
    "name": "Keeling",
    "firstName": "Patty",
    "note": 15
  },
  {
    "slug": "4febcf39-0304-4102-8cdf-dce4412f2da9",
    "name": "Bode",
    "firstName": "Audrey",
    "note": 14
  },
  {
    "slug": "f5d26aed-0f48-43d8-89f3-c9f4dcfb7b9d",
    "name": "Schamberger",
    "firstName": "Clifton",
    "note": 13
  },
  {
    "slug": "382f4d43-8e53-47d4-acac-71bafbfe2a2c",
    "name": "Willms",
    "firstName": "Carolyn",
    "note": 7
  },
  {
    "slug": "c3454c44-036b-4b40-ae0d-476dd396304d",
    "name": "Goldner",
    "firstName": "Hope",
    "note": 17
  },
  {
    "slug": "c3454c44-036b-4b40-ae0d-476dd396304d",
    "name": "Goldner",
    "firstName": "Hope",
    "note": 14
  },
  {
    "slug": "bbdf7566-234b-4b51-a50d-12e5d16258ea",
    "name": "Leffler",
    "firstName": "Herman",
    "note": 20
  },
  {
    "slug": "05f98060-71c8-4bc8-9941-d9e1ff22a709",
    "name": "Ledner",
    "firstName": "Leroy",
    "note": 19
  },
  {
    "slug": "bbdf7566-234b-4b51-a50d-12e5d16258ea",
    "name": "Leffler",
    "firstName": "Herman",
    "note": 14
  },
  {
    "slug": "f5d26aed-0f48-43d8-89f3-c9f4dcfb7b9d",
    "name": "Schamberger",
    "firstName": "Clifton",
    "note": 0
  },
  {
    "slug": "bbdf7566-234b-4b51-a50d-12e5d16258ea",
    "name": "Leffler",
    "firstName": "Herman",
    "note": 9
  },
  {
    "slug": "d53ce32f-ca6a-475c-9bb2-0736e16b558c",
    "name": "Altenwerth",
    "firstName": "Toni",
    "note": 10
  },
  {
    "slug": "382f4d43-8e53-47d4-acac-71bafbfe2a2c",
    "name": "Willms",
    "firstName": "Carolyn",
    "note": 11
  },
  {
    "slug": "4febcf39-0304-4102-8cdf-dce4412f2da9",
    "name": "Bode",
    "firstName": "Audrey",
    "note": 5
  },
  {
    "slug": "4febcf39-0304-4102-8cdf-dce4412f2da9",
    "name": "Bode",
    "firstName": "Audrey",
    "note": 3
  },
  {
    "slug": "7e5cc890-679f-4510-9dd8-d823f05c2dc3",
    "name": "Ernser",
    "firstName": "Julio",
    "note": 1
  },
  {
    "slug": "6138bc99-6599-42c1-a916-fb61d0672284",
    "name": "Lehner",
    "firstName": "Alonzo",
    "note": 12
  },
  {
    "slug": "c3454c44-036b-4b40-ae0d-476dd396304d",
    "name": "Goldner",
    "firstName": "Hope",
    "note": 2
  },
  {
    "slug": "4febcf39-0304-4102-8cdf-dce4412f2da9",
    "name": "Bode",
    "firstName": "Audrey",
    "note": 2
  },
  {
    "slug": "c3454c44-036b-4b40-ae0d-476dd396304d",
    "name": "Goldner",
    "firstName": "Hope",
    "note": 1
  },
  {
    "slug": "382f4d43-8e53-47d4-acac-71bafbfe2a2c",
    "name": "Willms",
    "firstName": "Carolyn",
    "note": 16
  },
  {
    "slug": "382f4d43-8e53-47d4-acac-71bafbfe2a2c",
    "name": "Willms",
    "firstName": "Carolyn",
    "note": 15
  },
  {
    "slug": "05f98060-71c8-4bc8-9941-d9e1ff22a709",
    "name": "Ledner",
    "firstName": "Leroy",
    "note": 17
  },
  {
    "slug": "4e74a047-70b5-4dde-8caa-d41c62f64e83",
    "name": "Keeling",
    "firstName": "Patty",
    "note": 18
  },
  {
    "slug": "4e74a047-70b5-4dde-8caa-d41c62f64e83",
    "name": "Keeling",
    "firstName": "Patty",
    "note": 17
  },
  {
    "slug": "4e74a047-70b5-4dde-8caa-d41c62f64e83",
    "name": "Keeling",
    "firstName": "Patty",
    "note": 11
  },
  {
    "slug": "bbdf7566-234b-4b51-a50d-12e5d16258ea",
    "name": "Leffler",
    "firstName": "Herman",
    "note": 19
  },
  {
    "slug": "4e74a047-70b5-4dde-8caa-d41c62f64e83",
    "name": "Keeling",
    "firstName": "Patty",
    "note": 4
  },
  {
    "slug": "d53ce32f-ca6a-475c-9bb2-0736e16b558c",
    "name": "Altenwerth",
    "firstName": "Toni",
    "note": 5
  },
  {
    "slug": "4febcf39-0304-4102-8cdf-dce4412f2da9",
    "name": "Bode",
    "firstName": "Audrey",
    "note": 11
  },
  {
    "slug": "4febcf39-0304-4102-8cdf-dce4412f2da9",
    "name": "Bode",
    "firstName": "Audrey",
    "note": 14
  },
  {
    "slug": "d53ce32f-ca6a-475c-9bb2-0736e16b558c",
    "name": "Altenwerth",
    "firstName": "Toni",
    "note": 11
  },
  {
    "slug": "4e74a047-70b5-4dde-8caa-d41c62f64e83",
    "name": "Keeling",
    "firstName": "Patty",
    "note": 18
  },
  {
    "slug": "4e74a047-70b5-4dde-8caa-d41c62f64e83",
    "name": "Keeling",
    "firstName": "Patty",
    "note": 6
  },
  {
    "slug": "6138bc99-6599-42c1-a916-fb61d0672284",
    "name": "Lehner",
    "firstName": "Alonzo",
    "note": 5
  },
  {
    "slug": "bbdf7566-234b-4b51-a50d-12e5d16258ea",
    "name": "Leffler",
    "firstName": "Herman",
    "note": 3
  },
  {
    "slug": "4febcf39-0304-4102-8cdf-dce4412f2da9",
    "name": "Bode",
    "firstName": "Audrey",
    "note": 15
  },
  {
    "slug": "bbdf7566-234b-4b51-a50d-12e5d16258ea",
    "name": "Leffler",
    "firstName": "Herman",
    "note": 15
  },
  {
    "slug": "6138bc99-6599-42c1-a916-fb61d0672284",
    "name": "Lehner",
    "firstName": "Alonzo",
    "note": 1
  },
  {
    "slug": "05f98060-71c8-4bc8-9941-d9e1ff22a709",
    "name": "Ledner",
    "firstName": "Leroy",
    "note": 0
  }
]
