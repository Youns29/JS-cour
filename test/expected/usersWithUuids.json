{
  "bbdf7566-234b-4b51-a50d-12e5d16258ea": [9, 19, 9, 12, 20, 14, 9, 19, 3, 15],
  "d53ce32f-ca6a-475c-9bb2-0736e16b558c": [
    13, 19, 10, 16, 8, 14, 19, 10, 5, 11
  ],
  "7e5cc890-679f-4510-9dd8-d823f05c2dc3": [10, 3, 1, 2, 18, 13, 18, 19, 18, 1],
  "05f98060-71c8-4bc8-9941-d9e1ff22a709": [19, 4, 12, 13, 15, 5, 3, 19, 17, 0],
  "382f4d43-8e53-47d4-acac-71bafbfe2a2c": [3, 17, 0, 17, 12, 18, 7, 11, 16, 15],
  "f5d26aed-0f48-43d8-89f3-c9f4dcfb7b9d": [1, 15, 19, 16, 20, 6, 14, 10, 13, 0],
  "6138bc99-6599-42c1-a916-fb61d0672284": [11, 6, 20, 10, 7, 13, 13, 12, 5, 1],
  "c3454c44-036b-4b40-ae0d-476dd396304d": [12, 4, 16, 13, 8, 20, 17, 14, 2, 1],
  "4e74a047-70b5-4dde-8caa-d41c62f64e83": [4, 5, 15, 15, 18, 17, 11, 4, 18, 6],
  "4febcf39-0304-4102-8cdf-dce4412f2da9": [20, 3, 7, 14, 5, 3, 2, 11, 14, 15]
}
